Looms.PoS/Looms.PoS.Application/Utilities/Helpers/TotalsHelper.cs

```diff
@@ -0,0 +1,217 @@
+using Looms.PoS.Domain.Daos;
+using Looms.PoS.Domain.Enums;
+using System.Net.WebSockets;
+
+namespace Looms.PoS.Application.Utilities.Helpers;
+
+// TODO: Uncomment when services have been added
+
+public static class TotalsHelper
+{
+    public static decimal CalculateOrderItemPrice(ProductDao? productDao, ProductVariationDao? productVariationDao, int quantity)
+    {
+        decimal price = 0;
+
+        if(productDao != null)
+        {
+            price = productDao.Price.Value * quantity;
+        }
+        else if(productVariationDao != null)
+        {
+            price = productVariationDao.Price.Value * quantity;
+        }
+        // else if (serviceDao != null)
+        // {
+        //     price = serviceDao.Price.Value * quantity;
+        // }
+
+        return price;
+    }
+
+    public static decimal CalculateOrderItemPrice(OrderItemDao orderItemDao, DiscountDao? discountDao, int quantity)
+    {
+        var price = orderItemDao.Price;
+        if(orderItemDao.Quantity != quantity)
+        {
+            price = CalculateOrderItemPrice(orderItemDao.Product, orderItemDao.ProductVariation, quantity);
+        }
+
+        if(discountDao != null)
+        {
+            price = CalculateTotalWithDiscount(discountDao, price);
+        }
+        return price;
+    }
+
+    public static decimal CalculateOrderItemTax(TaxDao tax, decimal price)
+    {
+        return Math.Round(price * Decimal.Divide(tax.Percentage, 100), 2, MidpointRounding.AwayFromZero);
+    }
+
+
+// This will calculation could cause minor discrepancies in the total amount due to rounding errors
+
+    public static decimal CalculateOrderTotal(OrderDao order)
+    {
+        decimal total = 0;
+        decimal tax = 0;
+        decimal price = 0;
+
+        if(order.OrderItems.Count == 0)
+        {
+            return total;
+        }
+
+        foreach (var orderItem in order.OrderItems)
+        {
+            price += orderItem.Price;
+            tax += orderItem.Tax;
+        }   
```

Comment:
`price = order.OrderItems.Select(x => x.Price).Sum()` same for tax

Reply:
added